src/Bundler.js

```diff
@@ -242,10 +245,24 @@ class Bundler extends EventEmitter {
 
         this.entryAssets = new Set();
         for (let entry of this.entryFiles) {
-          let asset = await this.resolveAsset(entry);
-          this.buildQueue.add(asset);
-          this.entryAssets.add(asset);
+          try {
+            let asset = await this.resolveAsset(entry);
+            this.buildQueue.add(asset);
+            this.entryAssets.add(asset);
+          } catch (err) {
+            throw new Error(
+              `Failed to load entrypoint: "${entry}" in "${
```

Comment:
I would say "Cannot resolve entry ${entry}". As of the error we haven't actually tried to load anything yet, just resolved.